osu.Game/Screens/Multi/Multiplayer.cs

```diff
@@ -32,8 +32,6 @@ public class Multiplayer : OsuScreen, IOnlineComponent
     {
         public override bool CursorVisible => (screenStack.CurrentScreen as IMultiplayerSubScreen)?.CursorVisible ?? true;
 
-        public override bool DisallowExternalBeatmapRulesetChanges => true;
```

Comment:
Why was this removed? This will cause the following:

```
System.InvalidOperationException: Attempted to lease a bindable that is already in a leased state.
   at osu.Framework.Bindables.Bindable`1.BeginLease(Boolean revertValueOnReturn)
   at osu.Game.Screens.OsuScreenDependencies..ctor(Boolean requireLease, IReadOnlyDependencyContainer parent) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/OsuScreenDependencies.cs:line 30
   at osu.Game.Screens.OsuScreen.createDependencies(IReadOnlyDependencyContainer dependencies) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/OsuScreen.cs:line 118
   at osu.Game.Screens.OsuScreen.CreateLeasedDependencies(IReadOnlyDependencyContainer dependencies) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/OsuScreen.cs:line 101
   at osu.Game.Screens.OsuScreenStack.screenPushed(IScreen prev, IScreen next) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/OsuScreenStack.cs:line 43
   at osu.Framework.Screens.ScreenStack.Push(IScreen source, IScreen newScreen)
   at osu.Framework.Screens.ScreenExtensions.<>c__DisplayClass0_0.<Push>b__0(ScreenStack stack)
   at osu.Framework.Screens.ScreenExtensions.runOnRoot(IDrawable current, Action`1 onRoot, Action onFail)
   at osu.Framework.Screens.ScreenExtensions.runOnRoot(IDrawable current, Action`1 onRoot, Action onFail)
   at osu.Framework.Screens.ScreenExtensions.Push(IScreen screen, IScreen newScreen)
   at osu.Game.Screens.Multi.Multiplayer.Start(Func`1 player) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Multi/Multiplayer.cs:line 164
   at osu.Game.Screens.Multi.Match.MatchSubScreen.onStart() in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Multi/Match/MatchSubScreen.cs:line 226
```

Reply:
Ah. This is unfortunate and untested..

Reply:
Fixed via #7741.